docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -309,16 +312,65 @@
 
       when: helmv3_installed and helmv3_dm_exists.rc == 0
 
-    - name: Install Deployment Manager
-      shell: KUBECONFIG=/etc/kubernetes/admin.conf /usr/sbin/helm upgrade --install deployment-manager {% if helm_chart_overrides is defined %}--values {{ helm_chart_overrides }}{% endif %} {{ manager_chart }}
-      when: helmv3_installed
-
-    # Install or Upgrade DM-Monitor
-    - name: "Install or Upgrade DM-monitor"
-      shell: "ansible-playbook {{ dm_monitor_playbook }}"
-      become: yes
-      ignore_errors: yes
-      when: dm_monitor_playbook
+    - name: Check deployment-manager App is present
+      shell: source /etc/platform/openrc; system application-list | grep deployment-manager | awk '{ print $10 }'
+      register: dm_app_status
+
+    # DM system app check, upload/apply helm-chart
+    - block:  
+        # Check dm-system app helm-chart
+       - name: Check DM system-app helm-chart present in /usr/local/share/applications/helm/
```

Comment:
Can we use an ansible variable instead of hard-coding the file path? So we can let the StarlingX user overwrite it if required